bin/clair-scan

```diff
@@ -30,7 +30,7 @@ function main
 {
     echo "-----------------------------------------------------------------------------------------"
     echo "Scanning of ${IMAGE}"
-    clair-scanner --report="${RESULTS_FILE}" --ip="${HOST_IP:-192.168.1.200}" --clair="http://${CLAIR_HOST_IP}:6060" "${IMAGE}" &>/dev/null
+    clair-scanner --whitelist=whitelist.yml --report="${RESULTS_FILE}" --ip="${HOST_IP:-192.168.1.200}" --clair="http://${CLAIR_HOST_IP}:6060" "${IMAGE}" &>/dev/null
```

Comment:
```suggestion
    clair-scanner --whitelist=../clair-whitelist.yml --report="${RESULTS_FILE}" --ip="${HOST_IP:-192.168.1.200}" --clair="http://${CLAIR_HOST_IP}:6060" "${IMAGE}" &>/dev/null
```

(and move the file to match this new name and location too).

Not a huge fan of non-executable files in a `bin/` folder.

Reply:
I agree. Should we make a new directory for config files or something?

Reply:
fixed.